### DISCUSSION TITLE
Is there a proper way to deploy an instance with ExternalDomain set to instance's fqdn OR ip address?

### DISCUSSION THREAD

**Discussion by UltrumDev** [zitadel/zitadel#6357] [Q&A] [answered]
Hi everyone! I want to deploy an instance of Zitadel but I'm unable to visit it outside of the machine Zitadel runs on. I'm using docker-compose deployment variant. I'm only able to visit the console from `localhost` address. Is there a way to access Zitadel server outside of this machine using dns name OR ip address? 
Getting `ID=QUERY-n0wng Message=Instance` not found error.
Thanks in advance

**Comment by fforootd** [comment, marked as answer]
Hi There

You need to instruct Zitadel about the point where it can be reached from the clients perspective.
Zitadel does accept ENV or Configfile to setup this.

Checkout these two links:

https://zitadel.com/docs/self-hosting/deploy/troubleshooting#instance-not-found
https://zitadel.com/docs/self-hosting/manage/custom-domain
